project-templates/node-json-theia/glsp-server/src/model/tasklist-model-index.ts

```diff
@@ -15,17 +15,27 @@
  ********************************************************************************/
 import { GModelIndex } from '@eclipse-glsp/server-node';
 import { injectable } from 'inversify';
-import { Task, TaskList } from './tasklist-model';
+import { Task, TaskList, Transition } from './tasklist-model';
 
 @injectable()
 export class TaskListModelIndex extends GModelIndex {
     protected taskIndex = new Map<string, Task>();
+    protected transitionIndex = new Map<string, Transition>();
```

Comment:
I think we should use a joint index all semantic elements here.
Otherwise adpopters that use this template as a starting point might pick up on an anitpattern and create a new index map for whenever they add a new element type to their semantic model.